README.md

```diff
@@ -15,6 +15,31 @@ A state controller with its own debugger
 ## Please head over to our WIP website for cerebral v2
 [http://www.cerebraljs.com/](http://cerebral.github.io/cerebral-website). You will find all the information you need there.
 
+## Contribute
+The entire Cerebral codebase has been rewritten to encourage contributions. The code is cleaned up, commented and all code is in a "monorepo". That means you can run tests across projects and general management of the code is simplified a lot.
+
+1. Clone the monorepo
+2. In root: `npm install` (Do not use YARN)
+3. Run `npm run configure` which will build code and bootstrap it together
+
+The packages are located under `packages` folder and there is **no need** to run `npm install` for each package.
+
+### Testing
+You can run all tests in all packages from root:
+
+`npm test`
+
+Or you can run tests for specific packages by going to package root and do the same:
+
+`npm test`
+
+### Changing the code
+When you make a code change you should create a branch first. When the code is changed and backed up by a test you can commit it from **the root** using:
+
+`git cz`
```

Comment:
Run `npm run configure` which will build code and bootstrap it together. Shouldn't it be npm run setup